core/amber/src/main/scala/edu/uci/ics/texera/web/resource/GoogleResource.java

```diff
@@ -0,0 +1,83 @@
+package edu.uci.ics.texera.web.resource;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.extensions.java6.auth.oauth2.AuthorizationCodeInstalledApp;
+import com.google.api.client.extensions.jetty.auth.oauth2.LocalServerReceiver;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.googleapis.auth.oauth2.GoogleClientSecrets;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.store.FileDataStoreFactory;
+import com.google.api.services.drive.Drive;
+import com.google.api.services.drive.DriveScopes;
+import com.google.api.services.sheets.v4.Sheets;
+import com.google.api.services.sheets.v4.SheetsScopes;
+import edu.uci.ics.texera.web.WebUtils;
+
+import java.io.*;
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import java.util.List;
+
+public class GoogleResource {
+    private static final String APPLICATION_NAME = "Texera";
+    private static final String TOKENS_DIRECTORY_PATH = WebUtils.getTokenPath();
+    private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+
+    /**
+     * Global instance of the scopes required by this quickstart.
+     * If modifying these scopes, delete your previously saved tokens/ folder.
+     */
+    private static final String CREDENTIALS_FILE_PATH = WebUtils.getCredentialPath();
+    private static final List<String> SCOPES = Arrays.asList(SheetsScopes.SPREADSHEETS, DriveScopes.DRIVE);
+
+    public static Sheets createSheetService() throws IOException {
+        try {
+            NetHttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
+            Credential credential = getCredentials(httpTransport);
+            return new Sheets.Builder(httpTransport, JSON_FACTORY, credential)
+                    .setApplicationName(APPLICATION_NAME)
+                    .build();
+        } catch (GeneralSecurityException e) {
+            e.printStackTrace();
+            return null;
+        }
```

Comment:
I think it's better to not handle (surpass) the exception, and let the framework handle it to report corresponding Response to frontend.  right now you are printing the error trace into stdout, which is not desirable.